src/Console/Processes/Process.php

```diff
@@ -324,4 +325,28 @@ protected function newSymfonyProcess($command, $path = null)
 
         return $process;
     }
+
+    /**
+     * Get process base path.
+     *
+     * @return string
+     */
+    public function getBasePath()
+    {
+        return preg_replace('/(.*)\/$/', '$1', $this->basePath);
+    }
+
+    /**
+     * Clone process from parent relative to base path.
+     *
+     * @return Process
+     */
+    public function fromParent()
+    {
+        $that = clone $this;
+
+        $that->basePath = str_finish(Path::resolve($this->basePath.'/../'), '/');
```

Comment:
Setting a protected property like this is allowed? Huh.

Reply:
Yeah weird that worked. Maybe because it's a clone of itself?